<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncidenceDateTimeEditor</class>
 <widget class="QWidget" name="IncidenceDateTimeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>191</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QCheckBox" name="mHasTimeCheck">
     <property name="whatsThis">
      <string>Set if this to-do's start and due dates have times associated with them.</string>
     </property>
     <property name="text">
      <string>Ti&amp;me associated</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="mStartLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Start:</string>
       </property>
       <property name="buddy">
        <cstring>mStartDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mStartCheck">
       <property name="whatsThis">
        <string>Sets the start date for this to-do</string>
       </property>
       <property name="text">
        <string extracomment="@option:check to-do start datetime">Sta&amp;rt:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KPIM::KDateEdit" name="mStartDateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string extracomment="@info:tooltip">Set the start date</string>
       </property>
       <property name="whatsThis">
        <string extracomment="@info:whatsthis">Sets the start date for this to-do</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPIM::KTimeEdit" name="mStartTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Sets the start time for this to-do.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mTimeZoneLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;hide&quot;&gt;&lt;font color='blue'&gt;&amp;lt;&amp;lt; Time zones&lt;/font&gt;&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mEndLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;End:</string>
       </property>
       <property name="buddy">
        <cstring>mEndDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mEndCheck">
       <property name="whatsThis">
        <string>Sets the due date for this to-do.</string>
       </property>
       <property name="text">
        <string extracomment="to-do due datetime">&amp;Due:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="KPIM::KDateEdit" name="mEndDateEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Sets the due date for this to-do.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPIM::KTimeEdit" name="mEndTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Sets the due time for this to-do.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPIM::KTimeZoneComboBox" name="mTimeZoneComboEnd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string extracomment="@info:whatsthis">Select the timezone for this event. It will also affect recurrences</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="mDurationLabel">
       <property name="text">
        <string>All day</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="mFreeBusyLabel">
       <property name="text">
        <string>Show time as:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mFreeBusyCombo">
       <property name="toolTip">
        <string extracomment="@info:tooltip">Set this event as Busy or Free time</string>
       </property>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox show event as busy time">Busy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox show event as free time">Free</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Recurrence:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="KComboBox" name="mRecurrenceCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mRecurrenceEditButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="mRecurrenceLabel">
       <property name="text">
        <string>Does not recur</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="KComboBox" name="mAlarmCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 minutes before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 minutes before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 minutes before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 minutes before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45 minutes before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 day before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 days before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 days before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 week before</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mAlarmEditButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reminder:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KTimeZoneComboBox</class>
   <extends>KComboBox</extends>
   <header>libkdepim/ktimezonecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPIM::KTimeEdit</class>
   <extends>QComboBox</extends>
   <header>libkdepim/ktimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPIM::KDateEdit</class>
   <extends>QComboBox</extends>
   <header>libkdepim/kdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
